examples/nextjs-v14-app/README.md

```diff
@@ -0,0 +1,9 @@
+# Lit in Next.js example
+
+This is a barebones demonstration of a Web Component authored with Lit working
+in a Next.js project with the App Router. You can also see an example of `@lit/react` usage.
+
+It uses the plugin from `@lit-labs/nextjs` to enable deep server rendering of
+Lit components.
+
+Note: Both bare custom elements and `@lit/react` wrapped components are placed in files with the `'use client';` directive. Deeply server rendering Lit components with declarative shadow DOM is not supported in React Server Components as React Flight server rendered data will contain the `<template>` element causing React hydration errors, and custom element definitions will not get included with client bundle.
```

Comment:
`React Flight server`? I'm not able to find what this is referring to.

Reply:
Answered in https://github.com/lit/lit/pull/4575#discussion_r1520451524
Will remove "Flight" from here too.